packages/components/src/TabBar/TabBar.component.js

```diff
@@ -117,11 +117,13 @@ function TabBar(props) {
 		</Tab.Content>
 	);
 	if (responsive && showDropdown) {
+		const selectedItem = items[selectedKey - 1];
 		return (
 			<React.Fragment>
 				<ActionDropdown
 					className={classnames(theme['tc-tab-bar-dropdown'], 'tc-tab-bar-dropdown')}
-					label={items[selectedKey - 1].label}
+					label={selectedItem.label}
+					icon={selectedItem.icon && selectedItem.icon.name}
```

Comment:
pls add "icon" to items prop-types shape. Test also would be nice to have.

Reply:
I added missed prop-type.
About test, I can't catch whith responsive behaviour even after using mount and adding wrapper with fixed width. It seems because of logic about calculation which depends on window and dom tree.
![image](https://user-images.githubusercontent.com/46000544/66314457-bd8cfb00-e91c-11e9-9944-2dc503c30c12.png)